index.html

```diff
@@ -45,6 +46,47 @@ <h1 class="title_hiring"> We're hiring </h1>
     </div>
     <!-- здесь заканчивается Колина часть html -->
 
+    <!--Nazar part-->
+
+    <div class="page-content">
+        <div class="header-first"><span>Engage Visitors</span></div>
```

Comment:
Do you need a `span` here?

Reply:
Or maybe `div` is not needed. No need to add an additional tag if thay do nothing

Reply:
Sorry i experimented with `span's` & `div's` and forgot delete. Was fixed